packages/framework-core/src/helpers/retrier.ts

```diff
@@ -0,0 +1,32 @@
+import { Class, Logger } from '@boostercloud/framework-types'
+
+export async function retryIfError<TReturn>(
```

Comment:
This function is the core of the optimistic concurrency solution. It retries a logic if a specific error is found